pyqtgraph/functions.py

```diff
@@ -1179,10 +1179,9 @@ def imageToArray(img, copy=False, transpose=True):
             # If this works on all platforms, then there is no need to use np.asarray..
             arr = np.frombuffer(ptr, np.ubyte, img.byteCount())
     
+    arr = arr.reshape(img.height(), img.width(), 4)
```

Comment:
I am struggling to understand this change.  Isn't this going to fail if you have an RGB array and try to force it from `rr x cc x 3` into `rr x cc x 4`?